src/app/organisms/room/RoomViewCmdBar.jsx

```diff
@@ -441,30 +441,33 @@ function RoomViewCmdBar({ roomId, roomTimeline, viewEvent }) {
     );
   }
 
+  const inSuggestionMode = cmd !== null;
+  const hasSuggestions = inSuggestionMode && (typeof cmd.suggestions === 'undefined' || cmd.suggestions?.length > 0);
```

Comment:
This change isn't strictly needed, but it made it much easier to understand the following section.